pkg/deployment/inventory.go

```diff
@@ -47,6 +47,8 @@ func GenerateNodeSetInventory(ctx context.Context, helper *helper.Helper,
 		return "", err
 	}
 
+	nodeSetGroup.Vars["edpm_enabled_services"] = instance.Spec.Services
```

Comment:
Services can be overridden in the deployment and this ansible var can have incorrect data, right?

Reply:
yeah, I'll close it for now